rosidl_generator_py/resource/_msg.py.template

```diff
@@ -53,4 +53,21 @@ class @(spec.base_type.type)(metaclass=@(spec.base_type.type)__Constants):
     @@@(field.name).setter
     def @(field.name)(self, value):
         self._@(field.name) = value
+@[if not field.type.is_primitive_type()]@
+        from @(field.type.pkg_name).msg import @(field.type.type)
+@[end if]@
+@[if field.type.is_array]@
+        from collections import Sequence
+        from collections import Set
+        from collections import UserList
+        from collections import UserString
+@[end if]@
+@[if field.type.type == 'byte']@
+        from collections import ByteString
+@[end if]@
+@[if field.type.type in ['char', 'string']]@
+        from collections import UserString
+@[end if]@
+        @(check_constraints(field.type))
```

Comment:
Instead of constructing this code in Python can this be done in the template instead? The tests should then not test the function but the expected behavior of the generated code.